source/docs/tools/index.md

```diff
@@ -4,11 +4,17 @@
 
 Each tool in Avogadro has an associated icon in the toolbar:
 
-![Toolbar](../../_static/toolbar_dark.svg)
+![The appearance of the toolbar in dark mode.](../../_static/toolbar_dark.svg)
 
-Hovering over a tool with the mouse gives a brief description of how to use it.
+Within Avogadro, hovering over a tool icon with the mouse gives a brief description of how to use it.
 
-After selecting a tool, the name of the tool and any options available for the tool are displayed in the [Tool Pane](panes-tool-pane).
+Tools can be selected either by left-clicking the icon, or using the keyboard by pressing <kbd>Ctrl</kbd> and the corresponding number key (1 through 9 and 0, left to right).
+
+After selecting a tool, the name of the tool and any options available for the tool are displayed to the left in the [Tool pane](panes-tool-pane).
+
+:::{versionadded} 2.0
+The Template Tool, Label Tool and Animation Tool are all completely new tools.
```

Comment:
These probably should be links to the relevant pages.

Reply:
Sure, I originally figured there was no need because the table of contents is right below it, but I had the same thought as you last night. I'll add it